domains/svijithprasad.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "svijithprasad",
+           "email": "",
+           "discord": "694159447790059521",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.G8V5ukgXfL16Z-fwAN4qDS4faEwK0WzXS60MuWwHYTgKolW4HjwT8hUBpK99JvKyNqB78bmrvbX_cKiq1yEop-907F_OYWaXFkFlM9FaLi4zb_gu_aRJ_ABmW4SD2hlUs5Lkq516cYhL-anGcsbW8jIharxrcCouPSUw0PwYZnc11E8NzhIRIN8m5sxQqj1TWP5L_Vc_Cr5yyvrRaWsN1p3-wep6gsIkAawchDXKDWfM9wjs2orrfOP0JQFLzKVKIM0e-nBlvqUdhFxqy473HucoMBAWi1fXsNZbD7SIUb2EMb8hOwX9YQT0fuB__IOMNMITDFxlyxNjqdf7lcJw7g.6EZGczxBeascAxNZhgbMrQ.YvwD-vK9j5FRIFSOKgvt0cBj0iOBVyJwMmQTDM-gjt6JYXzejxv_umlOkG7PfhrXgbcPfbPwH4VjK0M94wqlr1kVmgB3BQyLWPow9iwoGNM.DNOmvK0vCjtZQFhI3q9fDA"
+        },
+    
+        "record": {
+            "CNAME": "https://github.com/svijithprasad"
```

Comment:
Invalid CNAME record, must be a valid hostname without any protocols, file paths, and or URL parameters.